docs/administration/runner/runner-install.md

```diff
@@ -165,6 +165,103 @@ docker run -it \
 	 rundeckpro/runner:latest
 ```
 
+## Runner in Kubernetes
+
+These instructions will guide how to install a Runner in Kubernetes.
+
+1. Download a new Runner via API using the following **`curl`** request. Be sure to replace **`[URL]`** and **`[ApiToken]`** **`[ProjectName]`** with your Process Automation instance URL and API Token respectively:
+    ```
+    curl --location --request POST 'https://[URL]/api/42/runnerManagement/runners' \
+    --header 'Accept: application/json' \
+    --header 'X-Rundeck-Auth-Token: [ApiToken]' \
+    --header 'Content-Type: application/json' \
+    --data-raw '{
+    "name": "Kubernetes Runner",
+    "description": "Kubernetes runner",
+    "assignedProjects": {
+    "[ProjectName]": ".*",
+    },
+    "tagNames": "kubernetes"
+    }'
+    ```
+2. The response will provide a **`runnerId`** and **`token`**. Here is an example output:
+    ```
+    {"description":"Kubernetes runner","downloadTk":"d98d21bb-bc51-40ed-914a-85bcf618ffd0",
+    "name":"Kubernetes Runner","projectAssociations":{"Kubernetes":".*"},
+    "runnerId":"ad45e0c6-f6d3-4ca3-a0cf-044b4624fff3","token":"NopObWnk1MoeN0U7bqcfWTyavfwJzPTd"}
```

Comment:
Please make sure these are faked.  I like to always replace letters in the middle with a real word like "notarealtoken" or something to make it clear.